src/azul/plugins/metadata/hca/__init__.py

```diff
@@ -331,3 +336,6 @@ def service_config(self) -> ServiceConfig:
                 "protocol"
             ]
         )
+
+    def join_organism_age(self, value: Optional[str], unit: Optional[str]) -> Optional[str]:
+        return BaseTransformer.join_organism_age(value, unit)
```

Comment:
Calls undefined method. Stale changes from obsolete commits seem to be a recurring problem in your PRs. I always review all my changes before I commit them and scan all touched files for PyCharm warnings. This here would have been easy to catch because PyCharm warns about this line.